scripts/ansible/roles/windows/jenkins/tasks/environment-setup.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+---
+- name: Download all the installers
+  win_get_url:
+    url: "{{ item.value.url }}"
+    dest: "{{ item.value.dest }}"
+    timeout: 60
+  retries: 5
+  with_dict: "{{ packages }}"
+
+- name: Oracle JDK8 - Download
```

Comment:
I think that we can move the JDK8 install as part of the `jenkins/tasks/slave-setup.yml` since we don't need Java outside of the Jenkins context.

Reply:
Fixed